charts/tidb-operator/values.yaml

```diff
@@ -55,3 +55,11 @@ scheduler:
   kubeSchedulerImageName: gcr.io/google-containers/hyperkube
   # This will default to matching your kubernetes version
   # kubeSchedulerImageTag:
+
+admissionController:
+  create: false
+  replicas: 1
+  logLevel: 2
+  key: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBd3B5cmkvWnJJdS9TNEJlU05KZHBFN1JCeFF6MytaMTdxcXZWY2JweUY0cEZHdlMrCm1pWGF5ZDhVd3BYcDZJT3hDNDd3UWFhU1JnYm9qUTBYcjllWm1OS0NsWkxROUgwWVpTWndXSU4zL29vTEdWT3AKcDQ5ampBaDlFSmNhSTVFOUpNSlQvWFk1NHlZeTJ6a2lzaWhCUnI5b2FyMHZHZjdaeElQSGNndG5ES1ZFVm5vTgpJRGJuVTU2RE1GZk9kTWlvUURDU0ppakpPaU9pdXhFNTFVbG9aTVVVN3FCdTV5NUFtYTN2NVErd3dIR252ZUh1CmZ4TFpJN1plc3hhenlRMnU4SkFKZ1pIeWdEM1FTS1Z5REw4M1IyOHRmNDd0cm5QT1dOOXNreTNNNm9rZFZCaFQKUFVhUmRGMjkrWFFJdG56TG9Ua3RVSGtzdWVmTDF3dTNaVWdyVndJREFRQUJBb0lCQUF3bXRCOWRGM3RxOWdPLwo5UVVMbnlmM3pWTjhaODlTSWZLYUlBMmwwcEhkdEVybWxUQVN4UUp0eDJJTzhqZnVZUk9UclRTc0xkM3BlSU8yCkdaQ0ZLYlg5RHVJLzlJUVBlNEoxVC9aMjN2c3BxMSs5ZFk1Y3laNW91VVpiNk9ubm02Vzc3SHFhOHlwWHJEMDcKNkFxdnNNTzJGOUx5VlFxT2V4Sm1HQUhxN0ZoQnJWSk9QTS9lNWIydGl1SDFyUUJqVVhZWXAxVHp3R3BFM1cyQgpvRVI3bjlvNEJtNysyTnZ5bHAwNFdWWm9UNW5rRXVtcXB3ajR5NFdUYlBYNHFMUnJMVzJ4QnhnTzdUdVFQeVlFCkJUZFFZVkFhRm55akFXdXE3VDNETWJXVzQ3eDVnZ3BGR2hRNk9GYjRNUzN5RkpNZlE4eWhYblpOend6L1NvQXoKSWxub21pRUNnWUVBL2t0L0EreDhDcnZpWE1RR0duWVB6NW1sRmlmRW5FTEQwVkY0ak5pNFRWblBLYVhocEFCYgpNS3UwOXRuWXFnTHRacURBc0c4ZCtxWVlKOVZaQjMreUVnWTMwY043K1phVTlJcS81Q3R3N2p3d3JFMVEwTFN5CjhnQk9sZFVGUzVUMXFBOXhyOGx6WW1jdTl3TWpEZUNaTGxrdWdzWFlBRGF5eHRMRnpBU0kzWDBDZ1lFQXcrcTYKQnhGMjhkUFdwQk95bk5mcUIxdVc1Lyt6S3RhREh4Y1RMenRpaWdrM2NUbm1seTB0Zll4eExpeGs4aFVmV3dmQwoyejFDNmhxV1RFeS9nZTJHaWhlOFJDL1NFMGhvNEZyYUYwNVAvM0Voa0lwOFM1cWpobmdSdjJ2S1ZKU1BTLzNpCnJPRHBxNWxKU3pVc2lXWktTRFJ6RXlvRXI3Y2NodXpsNm8vYjFHTUNnWUFXMnE0bHFpRlR6YU1BZDFXZEZ6TkEKa3l4MUcwbmVTVm5UL0NCN2VCYklUT1FEZURjWlNEYnAzdVM2c3NWNE1oOG45QmFkUkw5TEFaWjJMZ1I4V2d1awo5dzRaT2hwd1NncEZHQnM2NUhTWDFLMERMMGV1R2k5aTlJZ2Jpek9Zb1dDYXBCaXhsY2RhOXE4UDRDWk1YdjBsClh6cUE0a3NSYWRrVGFXb0JkZjdlK1FLQmdRQ01PN0dZYVpuWUV5ZXFIaWRWT1A5YzhPUWRmOXVCS0RhR3hIK0kKNlZycEx4dXVRaUdaeUU2dGRSOTJTM1NlQ3h4OGpodWR6a1hkUjlhWkdyNWhnK0V2YXpucEhNbWozOHFoL0tYcQoxTFZTUkdvZ0NlQmFSWUFUT2xLSno5L3Rsc0RUUmhGQWFXVVkzUW1uZVpJOWJ5aHdjb3FFQnlsL2xQUW9aOTFTCmZsSjl2UUtCZ0dBTXRybTRFVkRTckF0QS9qeXg3NFB3SThjc2xtUEQ1d3JaWEZQVTUvc3FyRGNKbUNqSXBJRWUKRFdSaWNYL09aSUpRbDdjNkFOQjc1U1VUMHZ4dXJXeGp6WWlPVjRvUTJVUTBFc1VxcUh3RWxDVWZjclcxQnh2eQpmMnE3aGtUL0MySWwyRjhmRmxnYjJmZFVmaFM5MjZSbTF0aUUyMzJMU3MwYmZiWHArSDFmCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg==
```

Comment:
Why put the key here? This should be filled by the users, the default value is unnecessary. Just leave some documents in the comment to tell user how to fill this value.

Reply:
done to leave the documents in comment as a guider